onnx/defs/schema.h

```diff
@@ -612,6 +612,25 @@ class OpSchema final {
     return all_numeric_types_with_bfloat;
   }
 
+  static const std::vector<std::string>& all_numeric_types_with_bfloat_and_complex() {
+    static const std::vector<std::string> all_numeric_types_with_bfloat_and_complex = {
+        "tensor(uint8)",
+        "tensor(uint16)",
+        "tensor(uint32)",
+        "tensor(uint64)",
+        "tensor(int8)",
```

Comment:
What does it mean to have a SVD with integers? Eigen value will be likely be real even if inputs are integers tensors.

Reply:
Ah ok yes makes sense re: integers. Looks like the pytorch docs say the singular values will always be real. 

Is this the appropriate type signature then?

`T -> (T, S, T)  where T: tensor(float16) | tensor(float) | tensor(double) | tensor(bfloat16) | tensor(complex64) | tensor(complex128), S: tensor(float16) | tensor(float) | tensor(double) | tensor(bfloat16)`

Reply:
It works for me.

Reply:
done!